test/contracts/apps/recovery_to_vault.js

```diff
@@ -1,12 +1,15 @@
 const { hash } = require('eth-ens-namehash')
```

Comment:
It'd probably make sense to move all of these new tests to a new file (perhaps `test/contracts/depositable_delegate_proxy.js`), targeting just the DepositableProxy.

Reply:
Good idea. Reverted the changes in `recovery_to_vault.js` and created some specific tests for `DepositableDelegateProxy`.

Some of the things that we are testing here are already being tested in `recovery_to_vault.js`, `app_funds.js` and `kernel_funds.js`, so we could remove some tests from there.

Reply:
I wouldn't mind keeping those tests there; I view most of those tests as integration tests more than unit tests (pretty much only the bits in common are pure unit tests).